qa/tasks/cephfs/test_strays.py

```diff
@@ -1022,3 +1022,35 @@ def test_replicated_delete_speed(self):
 
         duration = (end - begin).total_seconds()
         self.assertLess(duration, (file_count * tick_period) * 0.25)
+    
+    def test_asok_dump_stray_command(self):
+        """
+        Test MDS asok dump stray command
+        """
+
+        LOW_LIMIT = 50
+        # need to create more folder to force fragmentation, creating more then needed
+        # to be on the safe side.
+        # we want to test the case when dumping stray folder must wait for the next dirfrag to be fetched
+        NUM_DIRS = LOW_LIMIT * 20
+        TOP_DIR = "topdir"
+        self.config_set("mds", "mds_bal_split_size", str(LOW_LIMIT))
+        self.assertEqual(self.config_get("mds", "mds_bal_split_size"), str(LOW_LIMIT), "LOW_LIMIT was not set on mds!")
+
+         # enable snaps
+        self.fs.set_allow_new_snaps(True)
```

Comment:
I guess this bit was taken from `test_snapshot_remove` test from this source. We need not explicitly enable snaps since it's enabled by default (test_snapshot_remove was introduced in 2017).